ts/src/base/Exchange.ts

```diff
@@ -4376,6 +4376,15 @@ export default class Exchange {
         return market;
     }
 
+    parseWsOHLCVs (ohlcvs: object[], market: any = undefined, timeframe: string = '1m', since: Int = undefined, limit: Int = undefined) {
```

Comment:
@samgermain I'm not sure about this. We have loads of exchanges using `parseOHLCV` instead of `parseWsoHLCV` (when the structure is the same, they call the rest parser. Second, upon returning to the watchOHLCV method, we have to do the `filterBYSinceLimit` again due to the newUpdates flag, so we would be doing it twice 
![image](https://github.com/ccxt/ccxt/assets/43336371/68397764-ec6f-4b9f-bdfe-79b446a58ec8)

Reply:
> @samgermain I'm not sure about this. We have loads of exchanges using `parseOHLCV` instead of `parseWsoHLCV` (when the structure is the same, they call the rest parser. Second, upon returning to the watchOHLCV method, we have to do the `filterBYSinceLimit` again due to the newUpdates flag, so we would be doing it twice ![image](https://user-images.githubusercontent.com/43336371/257052847-68397764-ec6f-4b9f-bdfe-79b446a58ec8.png)

We could write `parseWsOHLCV` to simply call `parseOHLCV` on the base exchange, and then exchanges that need a different implementation could do so?

Reply:
> Second, upon returning to the watchOHLCV method, we have to do the filterBYSinceLimit again due to the newUpdates flag, so we would be doing it twice 

The `parseOHLCVs` method calls `filterBySinceLimit` here, so I thought it made sense to do the same

Reply:
@samgermain the thing is, in the rest version it makes sense because all the information (since, limit, etc) is available right away inside `parseOHLCVs` whereas in the WS it is not, you just have access to it after the promise returns to the original execution context inside `watchOHLCV`  (after the watchTopics return in this case)

Reply:
> @samgermain the thing is, in the rest version it makes sense because all the information (since, limit, etc) is available right away inside `parseOHLCVs` whereas in the WS it is not, you just have access to it after the promise returns to the original execution context inside `watchOHLCV` (after the watchTopics return in this case)

ok, I just removed the `filterBySinceLimit` call inside `parseWsOHLCVs`, is that enough?

Reply:
@samgermain yes but should we keep the `return this.sortBy (results, 0) as any;` line?

Reply:
> @samgermain yes but should we keep the `return this.sortBy (results, 0) as any;` line?

it's up to you

Reply:
@samgermain I don't think it is needed